### PR TITLE
set ROLE_ADMIN to users with particular email address

### PR DESCRIPTION
**Pull request by alexandr-kalenyuk**
Part of implementation #4.

### CODE REVIEW

File: src/AppBundle/Security/User/UserProvider.php [hunk 1]
```diff
@@ -128,23 +128,21 @@ public function fetchData(AdUser $user, TokenInterface $token, adLDAP $adLdap)
...
         if ($userCollection) {
+            $user->setDisplayName($userCollection->displayName);
+            $user->setUuid($adLdap->utilities()->decodeGuid($userCollection->objectguid));
+            $user->setEmail($userCollection->mail);
+            $user->setPassword($token->getCredentials());
+
             $groups = $adLdap->user()->groups($user->getUsername(), $this->recursiveGrouproles);
             $sfRoles = array();
             $sfRolesTemp = array();
             foreach ($groups as $r) {
```
Comment (holyspecter) [inline src/AppBundle/Security/User/UserProvider.php]: what does `$r` mean? could you rename it?

File: src/AppBundle/Entity/User.php [hunk 2]
```diff
@@ -51,6 +44,16 @@ class User extends AdUser
      */
     private $roles = array();
 
+    /**
+     * @var array
+     */
+    public static $adminEmails = [
```
Comment (holyspecter) [inline src/AppBundle/Entity/User.php]: `private` seems more appropriate to me.

File: src/AppBundle/Security/User/UserProvider.php (hunk 1 above)
Comment (alexandr-kalenyuk) [inline src/AppBundle/Security/User/UserProvider.php]: sure. It's just piece of original code from ldap bundle

File: src/AppBundle/Entity/User.php (hunk 2 above)
Comment (alexandr-kalenyuk) [inline src/AppBundle/Entity/User.php]: matter of opinion

Comment (holyspecter) [inline src/AppBundle/Entity/User.php]: @alexandr-kalenyuk nope, putting it as `public` allows modifying it from any place of the project, which is not the behavior we want to have for temporary hardcoded list of admins.

Comment (Ishugaev) [inline src/AppBundle/Entity/User.php]: Why don't put it to config ?

Comment (Ishugaev) [inline src/AppBundle/Entity/User.php]: For my opinion holding of admin emails it is not responsibility of User Entity

**Comment by alexandr-kalenyuk** [comment]
@Ishugaev you are right it will better for design and testing. I will close this PR, because I have to admit that I don't have enough time to write full solution for this issue.

**Comment by holyspecter** [comment]
@alexandr-kalenyuk why have you closed this PR? This implementation is pretty ok for new, imo.

**Pull request closed without merging**